src/Questions.json

```diff
@@ -1,8 +1,7 @@
-  [
-  {
-  "question": "Как пройти в библиотеку?",
-  "answer": "Выйти из кабинета и следовать желтым стрелочкам",
-  "id": 1
+  [{

```

Comment:
It's not a good practice to name such files starting with a capital letter. It is not a React component.